app_check/testapp/Assets/Firebase/Sample/AppCheck/UIHandlerAutomated.cs

```diff
@@ -31,12 +31,20 @@ public class UIHandlerAutomated : UIHandler {
 
     private Firebase.Sample.AutomatedTestRunner testRunner;
 
+    // Your Firebase project's Debug token goes here.
+    // You can get this from Firebase Console, in the App Check settings.
+    private string appCheckDebugTokenForAutomated = "REPLACE_WITH_APP_CHECK_TOKEN";
+
     protected override void Start() {
+      // Set up the debug token, and install it as the factory to use.
+      DebugAppCheckProviderFactory.Instance.SetDebugToken(appCheckDebugTokenForAutomated);
+      FirebaseAppCheck.SetAppCheckProviderFactory(DebugAppCheckProviderFactory.Instance);
```

Comment:
Do we also need to initialize the firebase app?
like the InitializeFirebase method in UIHandler or otherwise referencing FirebaseApp.DefaultInstance

Reply:
When the tests start referencing DefaultInstance, that'll do any initialization logic (if is hasn't been done yet).